GenderPayGap.WebUI/Views/AdminOrganisationStatus/ChangeStatus.cshtml

```diff
@@ -0,0 +1,173 @@
+@using GenderPayGap.Core
+@using GenderPayGap.WebUI.Models.Admin
+@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model GenderPayGap.WebUI.Models.Admin.AdminChangeStatusViewModel
+
+@{
+    ViewBag.Title = "Change status - Administration - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@section BeforeMain {
+    @{
+        var crumbs = new List<CrumbViewModel>
+        {
+            new CrumbViewModel
+            {
+                Text = "Admin",
+                Href = Url.Action("Home", "Admin")
+            },
+            new CrumbViewModel
+            {
+                Text = Model.Organisation.OrganisationName,
+                Href = Url.Action("ViewOrganisation", "AdminViewOrganisation", new {id = Model.Organisation.OrganisationId})
+            },
+            new CrumbViewModel
+            {
+                Text = "Status history",
+                Href = Url.Action("ViewStatusHistory", "AdminOrganisationStatus", new {id = Model.Organisation.OrganisationId})
+            },
+            new CrumbViewModel
+            {
+                Text = "Change status"
+            }
+        };
+    }
+
+    @(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+    {
+        Crumbs = crumbs
+    }))
+}
+
+@{
+    var statusHints = new Dictionary<OrganisationStatuses, string>
+    {
+        {
+            OrganisationStatuses.Active,
+            "An organisation should be Active it is trading"
+        },
+        {
+            OrganisationStatuses.Retired,
+            "An organisation should be Retired if it has ceased trading. It will remain searchable and submissions will remain visible"
+        },
+        {
+            OrganisationStatuses.Deleted,
+            "An organisation should be Deleted if it was created by mistake / there was no reason for it to ever exist in the service"
+        },
+    };
+
+    var items = new List<ItemViewModel>();
+
+    foreach ((OrganisationStatuses status, string hint) in statusHints)

```

Comment:
😮 I didn't know this syntax exists - this is so much better than `KeyValuePair`!
This. changes. everything. 😀